src/scheduler/pbs_sched.c

```diff
@@ -1483,8 +1484,11 @@ main(int argc, char *argv[])
 		cmd = server_command(&runjobid);
 
 		if (connector >= 0) {
-			/* update sched object attributes on server */
-			update_svr_schedobj(connector, cmd, alarm_time);
+			if (update_svr) {
```

Comment:
You don't actually have to do this.  It is already done in update_svr_schedobj().  There is a static flag called 'svr_knows_me'.  Once that is set, you don't update anything.  If you want to do it here, remove the flag in update_svr_schedobj().

Reply:
Thanks for letting me know this @bhroam. I have removed the flag svr_knows_me and modified the code accordingly.